### PR TITLE
Fixed colorMode issue: Too many arguments passed!

### PR DESCRIPTION
**Pull request by aparms**
The `colorMode` issue is in khan/live-editor#576.

I only edited autosuggest-data.js The other files were changed during the build process.

### CODE REVIEW

File: build/js/live-editor.tooltips.js [hunk 1]
```diff
@@ -992,10 +992,10 @@ window.ScratchpadAutosuggestData = {
             description: i18n._("Calculates a color or colors between two color at a specific increment. The amount parameter is the amount to interpolate between the two values where 0.0 equal to the first point, 0.1 is very near the first point, 0.5 is half-way in between, etc."),
             params: [i18n._("c1: Interpolate from this color"), i18n._("c2: Interpolate to this color")]
         }, {
-            name: "colorMode(MODE)",
+            name: "colorMode(MODE, *range)",
             exampleURL: "/cs/colormode/5833774306689024",
             description: i18n._("Changes the way that color values are interpreted when set by fill()/stroke()/background()."),
-            params: [i18n._("MODE: Either RGB or HSB. The default is RGB.")]
+            params: [i18n._("MODE: Either RGB or HSB. The default is RGB."), i18n._("range (Optional): int or float: range for all color elements")]
```
Comment (kevinbarabash) [inline build/js/live-editor.tooltips.js]: Can you change the message to read `maxValue (Optional): number, maximum value for each color component`?

File: build/js/live-editor.tooltips.js [hunk 2]
```diff
@@ -992,10 +992,10 @@ window.ScratchpadAutosuggestData = {
             description: i18n._("Calculates a color or colors between two color at a specific increment. The amount parameter is the amount to interpolate between the two values where 0.0 equal to the first point, 0.1 is very near the first point, 0.5 is half-way in between, etc."),
             params: [i18n._("c1: Interpolate from this color"), i18n._("c2: Interpolate to this color")]
         }, {
-            name: "colorMode(MODE)",
+            name: "colorMode(MODE, *range)",
```
Comment (kevinbarabash) [inline build/js/live-editor.tooltips.js]: Can you change this to `"colorMode(MODE, *maxValue)"`?  I'm not sure why they use the word `range` in the official docs, but it seems confusing.

File: build/js/live-editor.ui.js [hunk 3]
```diff
@@ -7,7 +7,7 @@ this["Handlebars"]["templates"]["tipbar"] = Handlebars.template(function (Handle
 function program1(depth0,data) {
   
   
-  return "x";}
+  return "&times;";}
```
Comment (kevinbarabash) [inline build/js/live-editor.ui.js]: These are probably from a previous change where we forgot to include the compiled code.

**Comment by kevinbarabash** [comment]
@aparms thanks for the pull request.  You'll need to sign our contributor's license agreement before we can merge this.

www.khanacademy.org/r/cla

**Comment by aparms** [comment]
@kevinbarabash I signed the contributor license and (as you can see) I changed the function parameter and the help message. I also rebuilded.

**Comment by kevinbarabash** [comment]
@aparms live-editor is in maintenance mode now b/c of changing priorities so we're looking to limit the changes we make.

**Pull request closed without merging**